features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/utils/CacheUtils.kt

```diff
@@ -12,6 +12,10 @@ import androidx.collection.LruCache
 internal class CacheUtils<K : Any, V : Any>(
     private val invocationUtils: InvocationUtils = InvocationUtils()
 ) {
+
+    // some of this memory level are not being triggered after API 34. We still need to keep them for now
+    // for lower versions
+    @Suppress("DEPRECATION")
```

Comment:
what is the replacement then? should we use it for newer APIs?

Reply:
I think we can leave it like it is and prepare a task to remove those later. The only think that will happen for newer APIs is that those level will never be triggered. I don't see any harm now, maybe to create a task to clean these levels ?

Reply:
yes, no harm. My point is just that if something is deprecated, then probably there is a replacement for the newer APIs?

Reply:
Nope. no replacement suggested in their docs, I think they simply don't support these levels anymore and only kept few important ones. I think they did not need this granularity anymore.